stuffer/s2n_stuffer_text.c

```diff
@@ -227,6 +227,15 @@ static void s2n_va_list_cleanup(struct s2n_va_list *list)
     }
 }
 
+/* For CBMC verification, calls to "vsnprintf()" are re-directed to
+ * a local stub. For deployment, some hosts define "vsnprintf" as
+ * a macro that expands to call whatever is required, so for
+ * CBMC mode, we undef that symbol here
+ */
+#ifdef CBMC
+    #undef vsnprintf
+#endif
```

Comment:
If I don't undef it there, the _call_ to vsnprintf() in s2n_stuffer_text.c gets mangled by Apple's weird header files, so it doesn't call the stub at all, then the analysis fails... is there a better way to do this?

Reply:
Well... it means that you (i.e. developers) can run all the proofs locally if you have a macOS laptop in front of you, instead of waiting for a GitHub runner. It also means you can runs proofs on your local changes without needing a PR open, so you can check earlier and more often.

Reply:
Of course, it all works on an EC2 instance as well... so you should see 163 proofs succeed all the time. Without this fix, you get 162 successes and 1 failure on macOS... so we could revert this one change and open a separate issue for it if you prefer.

Reply:
I am happy to remove that "undef" from s2n_stuffer_text.c if you prefer, and raise a separate Issue for it. Your call... just let me know.